### DISCUSSION TITLE
SourceIP policy does not work via console

### DISCUSSION THREAD

**Discussion by g4-5u** [minio/minio#14515] [Q&A] [answered]
Hello, it seems that `aws:SourceIp` policy does not work when access from web console. A client (via browser) cannot list buckets when sourceip is client's IP, but can list when sourceip is server's own IP. Is this a correct behavior? Any workarounds?

https://docs.aws.amazon.com/AmazonS3/latest/userguide/example-bucket-policies.html#example-bucket-policies-use-case-3

**Comment by harshavardhana** [comment, marked as answer]
> Hello, it seems that `aws:SourceIp` policy does not work when access from web console. A client (via browser) cannot list buckets when sourceip is client's IP, but can list when sourceip is server's own IP. Is this a correct behavior? Any workarounds?

No workarounds - unfortunately, we can't support such use-cases at this moment.

**Reply by g4-5u** [reply]
Thank you for quick reply. I understand.
